libft/sources/map/mapss/ft_mapss_get.c

```diff
@@ -19,9 +19,9 @@ char	*ft_mapss_get(t_mapss *map, char *key)
 	if (!map || !map->head || !key)
 		return (0);
 	cur = map->head;
-	while (ft_strcmp(key, ((t_mapss_entry*)cur)->key) < 0)
+	while (cur && ft_strcmp(key, ((t_mapss_entry*)cur->content)->key) < 0)
```

Comment:
maybe we need to change here for > 0, i'm not sure, let me run one more test.
are you sure in this condition?

Reply:
`if (cur)` is basically the same as `if (cur > 0)` considering how 0/1/false/true are treated in C. Or you mean the strcmp result? Then definitely not, we scroll down the list WHILE what we have in hands is LESS than what we see in the catalogue. Once we hit >=, this means that either we've found what we were looking for, or we missed its place, so it was not in the list at all.

Reply:
Also if you're making tests, consider adding them to libft test suite, or at least listing here or in a separate issue. We are starting to lack tests sorely.

Reply:
Oops, right

Reply:
fixed